openreview/venue_request/process/recruitmentProcess.py

```diff
@@ -83,6 +83,12 @@ def process(client, note, invitation):
         error_status=f'''No recruitment invitation was sent to the following users due to the error(s) in the recruitment process: \n
         {recruitment_status.get('errors') }'''
 
-        comment_note.content['comment'] += f'''\n Error: {error_status}'''
+        comment_note.content['comment'] += f'''
```

Comment:
I don't think we need to change this file. The errors are correctly handled in the conference object.